hqdm/src/main/java/uk/gov/gchq/magmacore/hqdm/rdf/HqdmObjectFactory.java

```diff
@@ -136,7 +140,7 @@ private static <T extends Thing> java.lang.Class<T>[] irisToClasses(final Set<IR
     }
 
     // A statically initialized Map of IRIs to HQDM classes.
-    private static final Map<IRI, java.lang.Class<? extends Thing>> iriToClassMap = new HashMap<>(250);
+    private static final Map<IRI, java.lang.Class<? extends Thing>> iriToClassMap = new HashMap<>(400);
```

Comment:
Why was this increased to 400?  Was it to create headroom in case additional records are to be added?  If so, this still seems to be an arbitrary number.

Reply:
It's the starting capacity of the HashMap to prevent costly reorganisation of the data structure during its initial population. 250 was fine until we added the ability to extend HQDM, because extension modules can add to this map, and yes 400 is an arbitrary number which it probably has to be since we won't know if any extensions will be added or how many, so eventually the HashMap will have to reorganise itself anyway when there are many extension classes.